packages/grafana-ui/src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useContext } from 'react';
+import { css, cx } from 'emotion';
+import { stylesFactory, ThemeContext } from '../../themes';
+import { GrafanaTheme, locale } from '@grafana/data';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    counter: css`
+      label: counter;
+      border-radius: ${theme.spacing.lg};
+      background-color: ${theme.colors.bg2};
+      padding: ${theme.spacing.xxs} ${theme.spacing.xs};
+      color: ${theme.colors.textWeak};
+    `,
+  };
+});
+
+export interface CounterProps {
+  value: number;
+}
+
+export const Counter: FC<CounterProps> = ({ value }) => {
+  const theme = useContext(ThemeContext);
+  const styles = getStyles(theme);
+
+  return <span className={cx([styles.counter, 'counter'])}>{locale(value, 0).text}</span>;
+};
```

Comment:
I would move this into the tabs folder and make it an internal implementation detail of the tab component (So not expose it as public @grafana/ui component) . We have no other use case for this and as a general-purpose component, I do not think a counter component makes sense (it would be named badge / label in that case).

Reply:
Oh man, I totally messed up and completely misunderstood the task at hand. I'll remove the Counter component and push changes. 🙈